tests/integration/tier0/non-destructive/subscription-manager/test_sub_man_rollback.py

```diff
@@ -5,49 +5,23 @@
 from conftest import TEST_VARS
 
 
-@pytest.fixture(scope="function")
-def convert2rhel_repo(shell):
-    assert shell("rpm -qi subscription-manager").returncode == 0
-    # The following repository requires the redhat-uep.pem certificate.
-    # convert2rhel will try accessing the repo and during the test we run the convert2rhel twice
-    # making sure that the rollback of the first run correctly reinstalled the certificate
-    # when installing subscription-manager-rhsm-certificates.
-    c2r_repo = "/etc/yum.repos.d/convert2rhel.repo"
-
-    assert (
-        shell(
-            f"curl -o {c2r_repo} https://cdn-public.redhat.com/content/public/repofiles/convert2rhel-for-rhel-8-x86_64.repo"
-        ).returncode
-        == 0
-    )
-    assert os.path.exists(c2r_repo) is True
-
-    yield
-
-    assert shell(f"rm -f {c2r_repo}")
-    assert os.path.exists(c2r_repo) is False
```

Comment:
FYI for reviewers: This fixture was needed just for one of the cases tested by the below `test_sub_man_rollback` test. This case is no longer relevant because the convert2rhel repository no longer requires the /etc/rhsm/ca/redhat-uep.pem certificate to be in place.